server/lobbyconnection.py

```diff
@@ -938,7 +938,8 @@ async def launch_game(
             # options are. Currently, options for ladder are hardcoded into the
             # client.
             "name": game.name,
-            "init_mode": game.init_mode.value,
+            "game_type": GameType.to_string(self.game_type),
+            "init_mode": game.init_mode.value, # init_mode is deprecated
```

Comment:
You should add "use game_type instead" to the comment, and I'd put the comment on its own line (like the other comment)